validation/immersed_boundaries/Project.toml

```diff
@@ -1,3 +1,5 @@
 [deps]
+ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
```

Comment:
Ooh, I don't think we want these. Can we reset `Project.toml` and `Manifest.toml` to master?

Reply:
I thought the idea was to add things like this needed for the validation to the local project inside the validation, and not the main Project.toml?

Reply:
ah, I missed that this was in `validation/immersed_boundaries/Project.toml`! Apologies.